README.md

```diff
@@ -1,4 +1,33 @@
-# Azure API Manager Blue Green Deploy Function App
+# Azure Search Index Refresh Function App
+
+## Overview
+
+The purpose of this function app is to allow Azure Search indexes to be recreated without any down time. This is achieved by having a fixed endpoint for the search index managed as an API in Azure API Manager. Behind this there will exist an index in Azure Search which conforms to a naming convention captured by the following regular expression:
+
+``` RegEx
+(?<Name>.*)-(?<Version>[0-9]+-[0-9]+|)-(?<Deployment>a|b)-(?<Environment>dev|int|stag|prod)
+```
+
+### Example
+
+API Manager API
+
+* name: service-search-organisations
```

Comment:
Could the name be updated to something a bit more generic e.g. `my-api-name` with the external URL also being updated to something like `https://api.domain.name/my/api/name`. The reason for suggesting the changes is to avoid it being seen an actual real case and people getting hung up on it or thinking it would need to be updated should it change.